common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
+        <h1 class="title title--second">the NHS</h1>
```

Comment:
This is small thing and won't make any difference but why did you split this into two H1s?

As I understand, from SEO perspective it splits the weight of H1. It is ok to use multiple H1s in HTML5 but the recommendation is to use them as per block tag e.g. per section, not multiple H1s inside one section.

So wrapping the title--second in <span> inside <h1> should work better.